### ISSUE TITLE
Motion Load Failed

### ISSUE THREAD

**Issue by VRArt1** [kirurobo/UniWinApi#14]
I am attempting to use the pre-built vrm viewer however when I load in one of my models it shows them in t-pose and I receive a "Motion Load Failed" prompt under config. The head is tracking to the mouse but besides that my avatar is stuck in t-pose. How can I fix this?

**Comment by kirurobo** [comment]
Does it occur with other models like these?
https://3d.nicovideo.jp/search?word=vrm&page=1&limit=28&word_type=tag&category=all&work_type=vrm&download_filter=all&sort=created_at&order=1&perfect_match=0&usable_animation=

**Comment by VRArt1** [comment]
Tried one at random and it does work with one of those. Any clue as to what might be off with my vrm?

________________________________

**Comment by kirurobo** [comment]
Umm... If possible, would you please send me your VRM?
(Via DM on Twitter https://twitter.com/kirurobo , e.g.)

**Comment by somewqei** [comment]
Hello, I encountered the same issue with a character exported from VRoidStudio who is also stuck in Tpose. 
Expressions and bvh animations do not work on my character, but they work on characters from nicovideo and even other vroid Hub characters.

Was there a solution to this? Perhaps there is a problem with the set up inside VRoidStudio

**Comment by somewqei** [comment]
Ahh, I found the solution for VRoid Studio. When exporting the vrm, under Avatar Personality, The permission for 'who can assign a personality to the avatar?' needs to be set to 'Anyone'

Awesome app, thank you for providing this!

**Comment by kirurobo** [comment]
Thanks for your interest!
You are right, it is designed only T-pose without permissions.
It might be a good idea to show a message that it is due to permissions.

**Comment by somewqei** [comment]
Yes, a message would clear up confusion. Thanks again for sharing your code & example though. It's fun to see the potential of Unity!

**Comment by kirurobo** [comment]
[Ver. 0.7.0](https://github.com/kirurobo/UniWinApi/releases/tag/v0.7.0) allows motion to be applied regardless of "who can assign a personality to the avatar" permission.
This Issue is closed for now!
